GDFirebase/FirebaseAuth.gd

```diff
@@ -28,6 +28,8 @@ var refresh_request_body = {
     "refresh_token":""
     }
 
+# Sets the configuration needed for the plugin to talk to Firebase
+# These settings come from the Firebase.gd script
```

Comment:
Can we add "automatically" at the end of this? It's designed to function so that they don't have to do anything about this function, but I didn't make it start with an underscore because I don't like conventions like that (it's generally the convention in Godot that an underscore is meant to be private; it absolutely is not private, but it's meant to be).

Reply:
Added, makes sense